README.md

```diff
@@ -60,48 +71,76 @@ env:
+1. [Generate a new SSH key](https://help.github.com/articles/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent/).  
+    1. `ssh-keygen -t rsa -b 4096 -C "your_email@example.com"`
+    1. At the "Enter a file..." prompt, use: `deploy_key`
+    1. At the passphrase prompts, just hit Enter for no passphrase (twice).
+1. Add the public key (in `deploy_key.pub`) as a deploy key in your repository:
+    1. Go to `https://github.com/<proj-name>/<repo-name>/settings/keys` (You MUST be an admin to do this, or have an admin do these steps for you.)
+    1. Choose an appropriate title, e.g. "Travis Bikeshed Build Deploy"
+    1. Paste the contents of `deploy_key.pub`
+    1. Check the option to allow write access (if you don’t, Travis won’t be able to update the gh-pages branch.)
+    1. Delete the local file: `rm deploy_key.pub` (no longer needed)
 1. Use the Travis client to encrypt the deploy key
-  1. travis login --org
-  1. travis encrypt-file deploy_key
-  1. You don’t need to do any of the suggested items, but make a note of the openssl comment; it contains the encryption label that you need to add to .travis.yml.  For example, you might see something like:
+    1. `travis login --org` (and log in with github credentials, including 2FA)
+    1. `travis encrypt-file deploy_key`
+    1. You don’t need to do any of the suggested items, but make a note of the openssl comment; it contains the encryption label that you need to add to `.travis.yml`.  For example, you might see something like:
 ```
-     openssl aes-256-cbc -K $encrypted_0a6446eb3ae3_key -iv $encrypted_0a6446eb3ae3_key ...
+    openssl aes-256-cbc -K $encrypted_0a6446eb3ae3_key -iv $encrypted_0a6446eb3ae3_key ...
 ```
-  1. Check in “deploy_key.enc” to your repository.  You can remove “deploy_key” now if you like. Don’t check in “deploy_key”!
-1. Update .travis.yml by setting ENCRYPTION_LABEL with the encryption label printed when you encrypted the deploy key above. In the example, the encryption label from the openssl command is 0a6446eb3ae3.
+    1. Add the encrypted file to the repo: `git add deploy_key.enc`
+    1. Delete the key: `rm deploy_key` (don't check it in!)
+1. Update `.travis.yml` by setting `ENCRYPTION_LABEL` with the encryption label printed when you encrypted the deploy key above. In the example, the encryption label from the openssl command is `0a6446eb3ae3`.
 
 ## Finishing Up
-Check in the files created above.  In particular your “compile.sh”, “deploy.sh”, “.travis.yml”, and “depoy_key.enc” files should be checked in on the master branch.
+
+As a sanity check, run `git status` &mdash; you should have these new files staged in the `master` branch:
+
+* `.travis.yml`
+* `compile.sh`
+* `deploy.sh`
+* `deploy_key.enc`
+
+1. Commit these changes: `git commit -a -m "Prepare master branch for automatic build and deploy"`
+1. Publish the changes: `git push -f origin master`
+
+If it doesn't already exist, set up a `gh-pages` branch:
```

Comment:
I don't think this is necessary as comments in deploy.sh indicate that it will create one for you.  But you do need this to tell github to use the gh-pages branch for the docs.  Maybe add a comment above that deploy.sh will create one, but we're doing this manually here so we can tell github what to do.